uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/LoadLabUserTestRule.java

```diff
@@ -87,8 +95,23 @@ public void evaluate() throws Throwable {
         };
     }
 
+    private void createLabClient() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                BuildConfig.LAB_CLIENT_SECRET
+        );
+        mLabClient = new LabClient(authenticationClient);
+    }
+
     public String getLabUserUpn() {
         return upn;
     }
 
+    public LabAccount getLabAccount() {
+        return mLabAccount;
+    }
```

Comment:
Since we're now providing `getLabAccount` and therefore we may not need getLabUserUpn anymore